src/utils/Global.js

```diff
@@ -41,7 +43,14 @@ define(function (require, exports, module) {
     // inside Node for CI testing) we use this trick to get the global object.
     var Fn = Function, global = (new Fn("return this"))();
     if (!global.brackets) {
-        global.brackets = {};
+        // Earlier this initialization was happening inside appshell_extensions.js. But
+        // since the newer CEF versions have stronger JS checks render process was crashing
+        // siting JS eval error. So moved the initialization to here.
```

Comment:
nit: citing

Reply:
@petetnt Would you want me to add the link to `appshell_extensions.js` file here?

Reply:
I think that would be useful, yes!